test/hotspot/jtreg/runtime/os/TestHugePageDecisionsAtVMStartup.java

```diff
@@ -29,6 +29,7 @@
  * @requires os.family == "linux"
  * @modules java.base/jdk.internal.misc
  *          java.management
+ * @build jtreg.SkippedException
```

Comment:
I don't think these should be needed, should they?

Reply:
jtreg will build the class jtreg.SkippedException autimaticly, remove this directive will be okey. PR has been updated.